File: python/prism_sdk/federated_continual_resource_discovery_inference.py
"""Worldgen P05 AFA-worldgen-P05-F04 product wrapper."""
from .worldgen_resource_discovery_support import ResourceDiscoveryRequest, ResourceDiscoveryReceipt, manifest as _manifest, discover as _run
FEATURE_ID="AFA-worldgen-P05-F04"
CONTRACT_VERSION="worldgen-federated_continual-resource-discovery/1.0"
def worldgen_federated_continual_resource_discovery_inference_manifest(): return _manifest(feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, input_schema="ResourceDiscoveryRequest1@1", scale="federated continual autonomous", autonomy_tier="A1")
def discover_worldgen_federated_continual_resources(request): return _run(request, feature_id=FEATURE_ID, contract_version=CONTRACT_VERSION, scale="federated continual autonomous", require_federation=True)
__all__=["ResourceDiscoveryRequest","ResourceDiscoveryReceipt","worldgen_federated_continual_resource_discovery_inference_manifest","discover_worldgen_federated_continual_resources"]

File: crates/worldgen/src/federated_continual_resource_discovery_inference.rs
//! Worldgen P05 AFA-worldgen-P05-F04 federated_continual inference.
use super::resource_discovery_support::{self,ResourceDiscoveryRequest,ResourceDiscoveryReceipt};
pub const FEATURE_ID:&str="AFA-worldgen-P05-F04"; pub const CONTRACT_VERSION:&str="worldgen-federated_continual-resource-discovery/1.0";
pub fn worldgen_federated_continual_resource_discovery_inference_manifest()->serde_json::Value{resource_discovery_support::manifest(FEATURE_ID,CONTRACT_VERSION,"ResourceDiscoveryRequest1@1","federated continual autonomous","A1")}
pub fn discover_worldgen_federated_continual_resources(request:&ResourceDiscoveryRequest)->Result<ResourceDiscoveryReceipt,resource_discovery_support::ResourceDiscoveryError>{resource_discovery_support::discover(request,FEATURE_ID,CONTRACT_VERSION,"federated continual autonomous",true)}
pub use resource_discovery_support::{ResourceDiscoveryError,ResourceDiscoveryReceipt as Worldgenfederated_continualResourceinferenceReceipt,ResourceDiscoveryRequest as Worldgenfederated_continualResourceinferenceRequest};
